webpack.production.config.js

```diff
@@ -6,6 +6,12 @@ var WebpackCleanupPlugin = require('webpack-cleanup-plugin');
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 loaders.push({
+	test: /\.css$/,
```

Comment:
Can you instead include this in `webmaker.loaders.js` and remove the one from `webmaker.config.js` so that it can be used in both places?